### ISSUE TITLE
Still got the issue with the 0.0.15

### ISSUE THREAD

**Issue by mxmkmarquette** [o10n-x/wordpress-file-page-cache#7]
```
<br />
<b>Fatal error</b>:  Cannot declare class O10n\Filecache_Output, because the name is already in use in <b>/srv/users/serverpilot/apps/madmax/public/wp-content/plugins/wordpress-file-page-cache-master/output-cache.php</b> on line <b>15</b><br />
```

**Comment by mxmkmarquette** [comment]
if you want to reproduce the bug, just go https://max-mad.com/produit/coton-ouate-madmax-blanc  then do "ajouter au panier" it gonna fatal error.

after that if you go in the customize option for theme you gonna see the preview on the right is also in error mode.

**Comment by mxmkmarquette** [comment]
```
[26-May-2018 08:44:23 UTC] PHP Notice:  Undefined index: type in /srv/users/serverpilot/apps/madmax/public/wp-content/plugins/wordpress-javascript-optimization-master/controllers/js.class.php on line 1070
[26-May-2018 08:44:23 UTC] PHP Notice:  Undefined index: type in /srv/users/serverpilot/apps/madmax/public/wp-content/plugins/wordpress-javascript-optimization-master/controllers/js.class.php on line 1070
[26-May-2018 08:44:23 UTC] PHP Warning:  preg_match(): Unknown modifier '.' in /srv/users/serverpilot/apps/madmax/public/wp-content/plugins/wordpress-file-page-cache-master/controllers/filecache.class.php on line 291
[26-May-2018 08:44:23 UTC] PHP Warning:  preg_match(): Unknown modifier '.' in /srv/users/serverpilot/apps/madmax/public/wp-content/plugins/wordpress-file-page-cache-master/controllers/filecache.class.php on line 291
```
that what wp-debug told me.

**Comment by optimalisatie** [comment]
The issue should be solved since `v0.0.16`.

The regex notice indicates that you are using an invalid regular expression in the JSON config. The notice error in js.php indicates a similar JSON configuration related issue which should not be possible if the JSON is verified using the JSON schema.

**Comment by mxmkmarquette** [comment]
the error is fix in the 0.0.18 thank you very much for your hard work.
